code/controllers/subsystems/payment.dm

```diff
@@ -2,63 +2,116 @@ SUBSYSTEM_DEF(payment_controller)
+
+	var/list/brand = list(  //this is just for the jokes
+		"Cheesie Honkers",
+		"Men At Arms Tobacco",
+		"Lucky Stars Cigarettes",
+		"Carcinoma Angels Cigarettes",
+		"Robust Coffee",
+		"Getmore Chocolate Corp products",
+		"4no Raisins",
+		"SweatMax products",
+		"Space Cola",
+		"Space Mountain Wind",
+		"Dr.Gibb",
+		"ClothingLord 9000 Clothes",
+		"NanoTrasen Personal Computers",
+		"Rougelady Chewing Tobacco",
+		"Robust Softdrinks products"
+	)
+
+	var/list/slogan = list(
+		"Now with 18% less carcinogens!",
+		"Lawsuit free since 2579!",
+		"Leading doctors only moderately advise against our products!",
+		"Now occasionally without illegal narcotics!",
+		"Now complying with 94% of trading standards laws!",
+		"Condemned by the Space Pope!",
+		"Larva-tested, pupa-approved",
+		"For external use only",
+		"Fun for the whole family except grandma and grandpa",
+		"Beats a hard kick in the face",
+		"The proud result of prison labor!",
+		"No refunds",
+		"For the sophisticated shut-in",
+		"Tell your parents it's educational!"
+	)
+
+	//Don't forget our coporate overlords!
+	var/list/nanotrasen_nags = list(
+		"NanoTrasen staff are reminded that salaries cannot be re-negotiated once employment has commenced",
+		"NanoTrasen would like to remind staff that the Intellectual Property rights of any inventions and/or concepts created during your employment with NanoTrasen are waved in their entirety to NanoTrasen",
+		"Staff are reminded that unionizing is in direct violation of your employment contract. Dissatisfied staff are to book an appointment with a NanoTrasen Productivity and Morale Advisor, who will arrange an appointment with you within the next 895 working day(s)",
+		"NanoTrasen staff are reminded that minimising overhead and wastage to NanoTrasen is of paramount importance, as well as everyone's joint responsibility",
+		"NanoTrasen would like to remind contracted staff that failure to meet expectations during your annual performance review, can and will result in contract termination (termination fees applicable)",
+		"Staff are reminded that termination of the employment contract will incur a fee of 38.9% of your agreed annual salary for each year you have been employed by NanoTrasen, retrospectively adjusted for current inflation rates",
+		"NanoTrasen reserves the right to reimburse lost productivity caused by staff absence from the workplace for any reason (such as sickness), automatically from staff salary payment(s) until the lost sum is recovered in its entirety",
+		"Staff are once again reminded that comfort breaks are indeed on personal time, and as such will be docked from your pay entitlement"
+	)
```

Comment:
I had an idea for another. "NanoTrasen staff are reminded that payment processing fees are automatically deducted from your hourly pay entitlement, along with the mandatory convenience fees for the automatic deduction of the aforementioned processing fees"

Reply:
Love it